src/components/common/TmSelectNetwork.vue

```diff
@@ -46,6 +45,9 @@ export default {
       }
     }
   },
+  created: function() {
```

Comment:
this needs to be a watcher on networks. because networks might not be defined yet

Reply:
Why it could be undefined?

Reply:
And it's not triggered if networks were defined before TmSelectNetwork was loaded